### ISSUE TITLE
'Crash error 0' when trying to launch optifine on any version.

### ISSUE THREAD

**Issue by Geek1405** [sp614x/optifine#4946]
'Crash error 0' when trying to launch optifine on any version. i've tried removing forge, reinstalling optifine, and even reinstalling minecraft as a whole, and nothing, still the same error.

**Comment by kevorino** [comment]
I am also having this issue BUT not when launching. Only after a while of using shaders with OptiFine on version 1.16.3.

My computer's temperatures are good, I have a small factory motherboard boost for CPU but I've tried without.
I have my RAM clocked at 3600Mhz, and I've tried without.

I have tried multiple versions of Sildurs Shaders, latest version of KUDA shaders and latest of SEUS shaders.
Sildurs HIGH, EXTREME, EXTREME VL= Great FPS and last for a while then crashes if render distance is higher than 8 and in heavily edited region
KUDA = Not the best FPS, buggy and crashes sooner
SEUS = Good FPS but also crashes sooner

**CHANGES AFTER UPGRADE IN LINK DOWN BELOW**
- Game no longer freezes entire computer all the time (needing me to force restart via external button)

- "Successfully closes game" and gives error code 0

**YOU CAN READ THIS THREAD BELOW TO SEE EVERYTHING IVE DONE TO TRY TO FIX:**
[Old Issue Thread (may relate)](https://github.com/sp614x/optifine/issues/4288#issuecomment-713696512) 

Plz help and tell me I'm not going insane 🤪

**Comment by kevorino** [comment]
**Update:**
Issue persists with specific shaders under high (but not abnormal) loads, I believe its a GPU related issue.

I have GTX1080ti, I have tried RTX2070Super and gameplay felt smoother + no crashing. The 2070Super and 1080ti are relatively similar on performance.

I have ran multiple GPU benchmarks and stress tests and no crashes or "freezes".

Leads me to believe that my GPU has a fault or there is a issue with compatibility between game with shaders and GPU??

ugh...have to wait for 3080 to be available now...goodbye until then?
